docs/airgap.md

```diff
@@ -36,8 +36,16 @@ Create a config file like the following and save it to `config-save-bundles.yaml
 ```yaml
 source:
   repo:
+#   auth:    # Include auth if credentials for the registry are required.  Alternatively you can use environmental variables
+#     username: [USERNAME]
+#     password: [PASSWORD]  
     kind: HELM # or as any other supported Helm Chart repository kinds
     url: https://charts.trials.tac.bitnami.com/demo
+# containers:
+#   auth:
+#    registry: [URL]  # Do not include "https://" or trailing project and directory.  Eg. only include domain --> registry.pivotal.io
```

Comment:
Thanks for the contribution. I'd not add this comment since once we merge this fix https://github.com/vmware-tanzu/asset-relocation-tool-for-kubernetes/pull/146 the system will tell you the format of the domain name.